compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```
@@ -305,3 +305,4 @@ error.complex.variable.must.be.initialized=complex variable must be initialized
 error.isolated.var.cannot.be.declared.as.public=isolated variable cannot be public
 error.variable.declared.with.var.cannot.be.public=variable declared with var cannot be public
 error.field.binding.pattern.inside.list.binding.pattern=field binding pattern inside list binding pattern
+error.expression.expected.call.expression = invalid expression, expected call expression
```

I think this should be 
```suggestion
error.expression.expected.call.expression = invalid expression, expected a call expression
```